packages/lesswrong/components/recommendations/PostBottomRecommendations.tsx

```diff
@@ -136,16 +142,16 @@ const PostBottomRecommendations = ({post, hasTableOfContents, ssr = false, class
               }
               <AnalyticsContext pageSubSectionContext="curatedAndPopular">
                 {curatedAndPopularPosts?.map((post) => (
-                  <EALargePostsItem
+                  isEAForum ? <EALargePostsItem
                     key={post._id}
                     post={post}
                     className={classes.largePostItem}
                     noImagePlaceholder
-                  />
+                  /> : <PostsItem key={post._id} post={post} />
                 ))}
               </AnalyticsContext>
             </div>
-            <div className={classes.section}>
+            {isEAForum && <div className={classes.section}>
```

Comment:
You probably want to add a `skip` parameter to the hook that fetches this to avoid the extra DB query on LW